### ISSUE TITLE
Is it possible to build the cell with the following arguments? 

### ISSUE THREAD

**Issue by YuanbinLiu** [zhubonan/airsspy#3]
Is it possible to build the cell with the following arguments? If possible, a concrete example will be very helpful. Thanks a lot! 

#VARVOL=15
#SPECIES=A%NUM=4,B%NUM=1
#NFORM=2
#MINSEP=1.5

**Comment by zhubonan** [comment]
Yes. Those directives can be written directly in a `.cell` seed file as:

```text
#VARVOL=15
#SPECIES=A%NUM=4,B%NUM=1
#NFORM=2
#MINSEP=1.5
```

With `airsspy`, the equivalent global buildcell parameters are:

```python
from airsspy.seed import SeedAtoms

seed = SeedAtoms()
seed.build.varvol = 15
seed.build.species = "A%NUM=4,B%NUM=1"
seed.build.nform = 2
seed.build.minsep = 1.5

print(seed.build.to_string())
```

This serializes to:

```text
#VARVOL=15
#SPECIES=A%NUM=4,B%NUM=1
#NFORM=2
#MINSEP=1.5
```

The important detail is that `#SPECIES=A%NUM=4,B%NUM=1` is a cell-level `build.species` directive. It should be passed as the full buildcell syntax string, rather than as per-atom `SeedAtomTag.num` settings.
